### ISSUE TITLE
Unknown SwingMode state 2

### ISSUE THREAD

**Issue by ursusHU** [codyc1515/homebridge-panasonic-air-conditioner#27]
Since the last update one of my AC's throwing the following error:
2020. 06. 03. 8:38:49 [Bedroom Air Conditioner] Unknown SwingMode state 2
2020. 06. 03. 8:48:49 [Bedroom Air Conditioner] Unknown SwingMode state 2
2020. 06. 03. 8:58:49 [Bedroom Air Conditioner] Unknown SwingMode state 2
2020. 06. 03. 9:08:49 [Bedroom Air Conditioner] Unknown SwingMode state 2

I have the same AC in another room, which works fine, where the swing setting is set to forward and straight. In this case the Swing setting is up/down auto, and left side is set to right and right side to forward.

**Comment by efipso** [comment]
I have the same message, and also 'Unknown Temperature State 126 126':

[2020-6-4 6:52:47] [Panasonic AC] Unknown Temperature state 126 126
[2020-6-4 6:52:47] [Panasonic AC] Unknown SwingMode state 2
[2020-6-4 7:02:47] [Panasonic AC] Unknown Temperature state 126 126
[2020-6-4 7:02:47] [Panasonic AC] Unknown SwingMode state 2
[2020-6-4 7:12:47] [Panasonic AC] Unknown Temperature state 126 126
[2020-6-4 7:12:47] [Panasonic AC] Unknown SwingMode state 2
[2020-6-4 7:22:48] [Panasonic AC] Unknown Temperature state 126 126
[2020-6-4 7:22:48] [Panasonic AC] Unknown SwingMode state 2
[2020-6-4 7:32:48] [Panasonic AC] Unknown Temperature state 126 126
[2020-6-4 7:32:48] [Panasonic AC] Unknown SwingMode state 2
[2020-6-4 7:42:47] [Panasonic AC] Unknown Temperature state 126 126
[2020-6-4 7:42:47] [Panasonic AC] Unknown SwingMode state 2

Those messages doesn't appear when it's running, just when is switched off (standby)

**Comment by ursusHU** [comment]
The first message is definitely related to the directions, as soon as I turn it to auto mode it disappears. @codyc1515 If you want I can collect, and check all the possibilities and note the messages of your plugin.

**Comment by codyc1515** [comment]
It’s all good.

The swing mode thing just means the Swing Enabled indicator won’t show correctly for now. I have developed a fix for this. 

The temperature being 126 impacts nothing. It’s just some extra logging because the temperature always shows an invalid temperature at startup (126). I have turned down this logging now unless you have debug mode enabled.

There’s also an issue with the Eve / FakeGato recording. I’m testing a fix for this now too. 

The fixes for these should be pushed out tomorrow if all my testing goes well.
